Source/WebKit/WebProcess/Model/ModelProcessConnection.cpp

```diff
@@ -96,12 +98,14 @@ void ModelProcessConnection::invalidate()
 
 void ModelProcessConnection::didClose(IPC::Connection&)
 {
+    RELEASE_LOG_ERROR(Process, "%p - ModelProcessConnection::didClose", this);
```

Comment:
Is this really an error?

Reply:
Not really! Lemme change it to normal log.